### ISSUE TITLE
Changing WaveOutEvent volume changes master volume in Mixer / nerfs all other audio

### ISSUE THREAD

**Issue by nightblade9** [naudio/NAudio#668]
My game setup:

- I have sounds playing individually (usually just one at a time) via `SoundEffect.CreateInstance().Play`
- I have a single `WaveOutEvent` for my background audio
- I'm using `NAudio.Vorbis` to load the audio BGM file

Everything works fine. Until I try to set `Volume` on `WaveOutEvent`. Bad things happen:

- All game audio immediately becomes around 2/3 quieter
- In Windows Mixer, the volume of my app drops from 100 to 33

The perception of this in-game is that changing BGM volume changes all SFX volumes, too. 

You can see a tweet of this in action here: https://twitter.com/nightblade99/status/1283223485670719488/video/1

I tested it on two machines (Windows 10 and Windows 8.1) and saw similar behaviour.

How do I work around this / is it a bug? (It could be a MonoGame bug; playing multiple audio .wav files shows different issues.)

**Comment by markheath** [comment]
No, it's not a bug - it's calling a Windows API that sets the volume for that output device

A better approach is to actually change the volume of the samples you are sending to that device. E.g. If your SoundEffect instance has a volume property you can set it on that.

**Comment by nightblade9** [comment]
> A better approach is to actually change the volume of the samples you are sending to that device. E.g. If your SoundEffect instance has a volume property you can set it on that.

Is there a code sample for this? I thought `SoundEffect` and `SoundEffectInstance` don't work with OGG files. If they do, I don't know how to get that working.

**Comment by markheath** [comment]
`SoundEffect` is not part of NAudio, so I'm afraid I don't know what it's capabilities are
